### DISCUSSION TITLE
Interact with parent/preview directory directly?

### DISCUSSION THREAD

**Discussion by zackw** [alexluigit/dirvish#175] [Ideas]
### Thank you for the bug report

- [X] I am using the latest version of `dirvish` related packages.
- [X] I checked [CUSTOMIZING|EXTENSIONS](https://github.com/alexluigit/dirvish#resources).
- [ ] You may also try reproduce the issue using clean environment and minimal configurations with
the command `emacs -Q`.

### Bug description

In the multi-pane display brought up by `M-x dirvish`, weird things happen if you try to interact with the left (parent directory) pane at all, or the right (preview of selected directory entry) pane while that pane is displaying a subdirectory.  Undesirable "weird things" I have noticed in 20 minutes of fooling around with it:

- Normal "window" selection operations won't let me put keyboard focus into the left pane. This might be intentional but it doesn't feel Emacsy.  I expected to be able to `C-x o` in there and then interact with it like it was the middle (current directory) pane.
- Clicking on items in the parent pane with left or right mouse button does nothing. I expected this to load the selected directory in the middle pane.
- Clicking on items in the parent pane with middle mouse button mangles the contents of the parent pane -- inserting random gibberish, possibly whatever happens to be in X PRIMARY selection at the time -- and _does_ put keyboard focus into the left pane, at which point it becomes apparent that _that pane is editable like it's in Fundamental mode_, which is certainly not what you meant to happen.
- `C-x o` _will_ put me into the preview pane.  If this pane is previewing a regular file, this lets me interact with it like I'd visited the file normally -- this is Good.  However, if this pane is previewing a subdirectory, its contents _appear_ to be a Dired buffer but _it's actually in Fundamental mode_ (unlike the parent pane it genuinely is in that mode according to `C-h m`) and therefore does not behave anything like I expect a preview of a directory to behave.

### OS

Linux

### Emacs Version

27

**Comment by alexluigit** [comment]
Hi @zackw , thanks for your report.  In the early versions of dirvish, actual dired buffers were used to fill the parent/preview window, but eventually this approach was dropped because of some issues it introduces:
1. it is hard to keep track of different settings (header, file details, etc.) for different dired buffers
2. it populates the buffer list with a lot of meaningless buffers
3. these extra buffers caused a lot of troubles on session reuse/init

For these reasons, the content in the parent/preview windows in dirvish now are just colorized plain text.  If you want to interact with parent/preview directory, you can just navigate into that directory first.
